t/cobrand/closest.t

```diff
@@ -72,8 +72,9 @@ FixMyStreet::override_config {
     $near = $c->find_closest_address_for_rss($report);
 
     ok !$near, 'no closest address for RSS if not cached';
+
+    my $json = $mech->get_ok_json('/ajax/closest?lat=55&lon=-1');
```

Comment:
This will use the `Default` cobrand, not `UK` like the rest of the tests, as `FixMyStreet::override_config` doesn't set `ALLOWED_COBRANDS`.

Reply:
Added another UK-test.